httpx/concurrency/asyncio.py

```diff
@@ -176,13 +166,13 @@ def loop(self) -> asyncio.AbstractEventLoop:
                 self._loop = asyncio.new_event_loop()
         return self._loop
 
-    async def connect(
+    async def open_tcp_stream(
```

Comment:
I don't think the symmetry of this method name with the backend serves us well. Can we revert to `connect()`?

Reply:
I guess you’re referring to the symmetry with TCPStream?

I’m considering what will happen when we want to connect via UDP here. Instead of having a broad connect() method and a more precise connect_udp() or similar, I thought we could name this method after what it *actually* does: open a Stream that happens to represent a TCP connection. And then we can have open_udp_stream(), and all things will be crystal clear.

Would you reconsider it under this angle?

Reply:
For some reason I thought this change was on the stream interface. Disregard my comment. :)